packages/serverpod_test/lib/serverpod_test_public_exports.dart

```diff
@@ -0,0 +1,14 @@
+library serverpod_test;
+
+export 'serverpod_test.dart'
```

Comment:
Suggestion: Flip the naming for the public and the internal exports.

We probably want the name for the export our users should be using to be as simple as possible. Then we can call the other one `_internal_exports`.

Reply:
The strategy is that people should really never import anything directly from `serverpod_test`. They should always import from the generated file. We're just re-exporting this file in the generated file and in that sense I think this is very explicit.